kale/prepdata/tabular_transform.py

```diff
@@ -0,0 +1,92 @@
+"""
+Functions for manipulating tabular data
+"""
+
+import os
+from typing import Any, Dict, List, Tuple
+
+import pandas as pd
+
+
+def apply_confidence_inversion(data: pd.DataFrame, uncertainty_measure: str) -> Tuple[Any, Any]:
+    """Invert a list of numbers, add a small number to avoid division by zero.
+
+    Args:
+        data (Dict): Dictionary of data to invert.
+        uncertainty_measure (str): Key of dict to invert.
+
+    Returns:
+        Dict: Dictionary with inverted data.
+    """
+
+    if uncertainty_measure not in data:
+        raise KeyError("The key %s not in the dictionary provided" % uncertainty_measure)
+
+    # Make sure no value is less than zero.
+    min_not_zero = min(i for i in data[uncertainty_measure] if i > 0)
+    data.loc[data[uncertainty_measure] < 0, uncertainty_measure] = min_not_zero
+    data[uncertainty_measure] = 1 / data[uncertainty_measure] + 0.0000000000001
+    return data
+
+
+def get_data_struct(
```

Comment:
This name is very generic but the actual functions are not. A more specific name will be better.